test/e2e/e2e_suite_test.go

```diff
@@ -24,118 +24,98 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"os/exec"
-	"path"
 	"testing"
-	"time"
-
-	"github.com/onsi/ginkgo"
-	"github.com/onsi/gomega"
-	appsv1 "k8s.io/api/apps/v1"
-	apimachinerytypes "k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/cluster-api/test/helpers/kind"
-	"sigs.k8s.io/cluster-api/test/helpers/scheme"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	capiv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/test/framework"
+	"sigs.k8s.io/cluster-api/test/framework/generators"
+	"sigs.k8s.io/cluster-api/test/framework/management/kind"
 	"sigs.k8s.io/cluster-api/util"
-	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func TestE2E(t *testing.T) {
-	gomega.RegisterFailHandler(ginkgo.Fail)
-	ginkgo.RunSpecs(t, "e2e Suite")
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Cluster API E2E Suite")
 }
 
 const (
 	capiNamespace      = "capi-system"
+	capiTestNamespace  = "default"
 	capiDeploymentName = "capi-controller-manager"
 	setupTimeout       = 10 * 60
 )
 
 var (
-	managerImage    = flag.String("managerImage", "", "Docker image to load into the kind cluster for testing")
-	capiComponents  = flag.String("capiComponents", "", "URL to CAPI components to load")
-	kustomizeBinary = flag.String("kustomizeBinary", "kustomize", "path to the kustomize binary")
```

Comment:
Have you seen how the e2es are running on prow? Will we need to provide these features to the framework? namely being able to override various paths/binaries?

Reply:
Which e2es are these? For the CAPI e2e tests we should just need to run `make test-e2e`. The manager image is being passed in via an env var. 
If we need access to other things for prow/CI, we can add those in as part of #1884 
WDYT?

Reply:
Oh got it, i thought these were already running on prow as jobs